.github/ISSUE_TEMPLATE/tracking.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "📊 Tracking Issue"
+title: "📊Tracking: [service]"
+about: Add a module tracking issue (for project management)
+labels: management/tracking
+---
+
+### Add your +1 👍  to help us prioritize high-level constructs for this service
+--- 
+
+### Overview:
+<!-- 
+Summary of the service (leverage the service’s product page for the text) and a link to the relevant AWS Docs. This should be the same text that we put at the top of the package’s README.db. Also include a link to the service’s CDK Construct Library API reference page. 
+-->
+
+
+
+
+[AWS Docs](url)  
+[CDK API Reference](url)
+
+### Implementation:
+<!-- 
+Checklist of use cases, constructs, features (such as grant methods) that will ship in this package 
+(not required until the issue is added to the public roadmap)
+-->
+
+
+
+
+### Related Issues:
+<!-- 
+e.g. checklist of links to feature requests, bugs, and PRs that are in scope for GA release of this module 
+(not required until the issues is added to the public roadmap)
+-->
+
+
+
+
+<!-- 
+Labels to add:
+    - management/tracking
```

Comment:
I believe it's possible to configure this so labels are automatically added to any new issue opened here.